src/views/mfa-verify/PushForm.js

```diff
@@ -29,6 +31,18 @@ define(['okta'], function (Okta) {
 
     initialize: function () {
       this.enabled = true;
+      this.listenTo(this.options.appState, 'change:isSuccessResponse',
+        function (state, isSuccessResponse) {
+          if (isSuccessResponse && this.settings.get('features.autoPush')) {
+            var autoPush = $('.custom-checkbox [name=autoPush]');
+            if (autoPush[0].checked) {
```

Comment:
Continuing on that thread, we're generally not working with the dom elements themselves - those views are usually tied to a model, and we can access the model to get the current value. Check out Ujjwal's last pr for rememberDevice:
https://github.com/yumingcao-okta/okta-signin-widget/commit/7ad708db07845d89a8ad2cb84fac1d424b3a57e0